### PR TITLE
Fetching employees data

### PR DESCRIPTION
**Pull request by vrackovic**
- Bootstrapped `redux` and `redux-saga` files so that employees are fetched and saved to store
- Employees displayed in Employees component and styles as suggested in design mocks

NOTE: Merge after #136

### CODE REVIEW

File: src/main/frontend/src/components/employees/EmployeeInfo.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  align-items: center;
+  font-size: 14px;
+`;
+
+const EmployeeInfo = employee => {
+  const { fullName, email, jobTitle, yearsOfService } = employee;
+  return (
+    <EmployeeContainer>
+      {/* User picture should be rendered here */}
+      <div
```
Comment (iscekic) [inline src/main/frontend/src/components/employees/EmployeeInfo.js]: Use a styled component

File: src/main/frontend/src/components/employees/EmployeeInfo.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        }}
+      >
+        <EmployeesImage />
+        <EmployeesName>{fullName}</EmployeesName>
+      </div>
+      <EmployeesAdditionalInfo>
+        <p>{jobTitle}</p>
+        <p>{email}</p>
+        <p>Years of Service: {yearsOfService}</p>
+        <Button click={() => {}} title="View Profile" />
```
Comment (iscekic) [inline src/main/frontend/src/components/employees/EmployeeInfo.js]: If this no-op is intended make a `TODO` comment

File: src/main/frontend/src/components/employees/Employees.js [hunk 3]
```diff
@@ -1,9 +1,40 @@
...
 
 class Employees extends Component {
+  componentDidMount() {
+    this.props.requestEmployees();
+  }
   render() {
-    return <div>Employees</div>;
+    return (
+      <EmployeesContainer>
+        {this.props.employees.length > 1 &&
```
Comment (iscekic) [inline src/main/frontend/src/components/employees/Employees.js]: Make initial `employees` state an empty array and you can remove this check (looks like you already did this).

File: src/main/frontend/src/store/services/employeesService.js [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
-};
+import request from 'superagent';
+
+export const getEmployeesApi = async () => {
+  try {
+    return await request
+      .get('/users')
+      .set('X-ID-TOKEN', sessionStorage.getItem('token'))
+      .then(res => res.body)
+      .catch(err => {
```
Comment (iscekic) [inline src/main/frontend/src/store/services/employeesService.js]: Unneeded `catch`

File: src/main/frontend/src/components/employees/Employees.js (hunk 3 above)
Comment (vrackovic) [reply src/main/frontend/src/components/employees/Employees.js]: I think i removed this in next PR :smile:

**Pull request merged**